challengeMk2/Views/SystemDetailCarouselPage.xaml

```diff
@@ -0,0 +1,188 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:skip="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:cvt="clr-namespace:challengeMk2.Converters"
+             xmlns:slt="clr-namespace:challengeMk2.DataSelectors"
+             mc:Ignorable="skip"
+             x:Class="challengeMk2.Views.SystemDetailCarouselPage"
+             Title="{Binding Title}">
+    <ContentPage.ToolbarItems>
+        <ToolbarItem IconImageSource="tab_star.png"/>
+    </ContentPage.ToolbarItems>
+
+    <ContentPage.Resources>
+        <ResourceDictionary>
+
+            <!--GENERAL INFO-->
+            <DataTemplate x:Key="GeneralTemplate">
```

Comment:
Would be nice to extract the inline templates into separate XAML components and refer to them here.